guides/common/modules/proc_configuring-satellite-to-use-external-databases.adoc

```diff
@@ -21,10 +21,10 @@ Use the `{foreman-installer}` command to configure {Project} to connect to an ex
   --katello-candlepin-db-name candlepin \
   --katello-candlepin-db-password _Candlepin_Password_ \
   --katello-candlepin-manage-db false \
-  --katello-pulp-db-username pulp \
-  --katello-pulp-db-password _pulp_password_ \
-  --katello-pulp-db-seeds _mongo.example.com:27017_ \
-  --katello-pulp-db-name _pulp_database_
+  --foreman-proxy-content-pulpcore-manage-postgresql false \
+  --foreman-proxy-content-pulpcore-postgresql-host _postgres.example.com_ \
+  --foreman-proxy-content-pulpcore-postgresql-db-name pulpcore \
+  --foreman-proxy-content-pulpcore-postgresql-password _Pulpcore_Password_
```

Comment:
Do we want to show the other options, such as any of these?:

```
--foreman-proxy-content-pulpcore-postgresql-user  User of the Pulpcore PostgreSQL database.
--foreman-proxy-content-pulpcore-postgresql-port  Port of the Pulpcore PostgreSQL database.
--foreman-proxy-content-pulpcore-postgresql-ssl  Enable SSL connection to the Pulpcore PostgreSQL database. Only meaningful for external/unmanaged DB.
--foreman-proxy-content-pulpcore-postgresql-ssl-cert  Path to SSL certificate to use for Pulpcore connection to PostgreSQL database.
--foreman-proxy-content-pulpcore-postgresql-ssl-key  Path to key file to use for Pulpcore connection to PostgreSQL database.
--foreman-proxy-content-pulpcore-postgresql-ssl-require  Configure Pulpcore to require an encrypted connection to the PostgreSQL database.
--foreman-proxy-content-pulpcore-postgresql-ssl-root-ca  Path to the root certificate authority to validate the certificate supplied by the PostgreSQL database server.
```

Reply:
We probably ought to do it for all the external databases if we want to encourage the use of things like the SSL parameters. I'd like to keep this as is to follow the other examples and create an issue to explore